src/net/sourceforge/kolmafia/swingui/OptionsFrame.java

```diff
@@ -1628,7 +1628,8 @@ public void update() {
                   {"allowCloseableDesktopTabs", "Allow tabs on main window to be closed"},
                 };
 
-    private final JComboBox looks, toolbars, toolIcons, scripts;
+    private final JComboBox<Object> looks;
```

Comment:
Can this be JComboBox<String>? It looks like we either add the constant string "Always use OS default look and feel" or we add installed[i].getName().